src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -404,6 +405,7 @@ public partial class BadImageFormatException : System.SystemException
     }
     public partial struct Boolean : System.IComparable, System.IComparable<bool>, System.IConvertible, System.IEquatable<bool>
     {
+        private object _dummy;
```

Comment:
Why Boolean contains an object field?
The real one only contains a bool.

Reply:
Just pushed a change that fixes this.